documentation/cloud-manager-guide/admin/user-management/edit-user.md

```diff
@@ -0,0 +1,122 @@
+---
+keywords:
+title: Edit a User
+description: Edit a user
+---
+# Edit a User
+
+This guide outlines the required steps to edit the details, status, role, and environment assignments for a specific user in Environment Operations Center.
+
+## Getting started
+
+There are two ways to open the *Edit User* tab, either by selecting the user's name or by selecting the **Options** (**...**) menu associated with that user.
+
+To access the *Edit User* tab by directly selecting the user, select the user name from the list of users on the *Admin* *User* tab.
```

Comment:
I think an H3 would be good here to say "Edit a user using Options tab" and then another after for "Edit a user by selecting a user" (but better wording than that!)

Let's reverse these and put the Options (...) menu first because it's only one screen and then put the clicking on a user and editing second as it's two screens.